doc/sal_constraints_and_recommendations.rst

```diff
@@ -383,6 +393,8 @@ If a time-of-data is to be associated with an item it should be named
 
 The time(s) should be obtained using the SAL getCurrentTime() method, which returns a double precision value of TAI time with a resolution of at least 0.001 seconds.
 
+.. _sal_constraints_and_recommendations:ignored-attributes-in-topics:
```

Comment:
In the paragraph "The time(s)..." please consider "value of TAI time in unix seconds, with a resolution..." (add "in unix seconds")